### ISSUE TITLE
uberon and Wikidata

### ISSUE THREAD

**Issue by ChristianKleineidam** [obophenotype/uberon#1235]
Can uberon be imported in Wikidata?

**Comment by cmungall** [comment]
I believe this is being worked on cc @elviram

**Comment by elviram** [comment]
Yes it is. If anybody wants to offer help or expertise I can give them a crash curse on Wikidata.

**Comment by cmungall** [comment]
Follow up on general issues here: https://github.com/OBOFoundry/OBOFoundry.github.io/issues/285

@elviram - I like the idea of a "crash curse" 😉

**Comment by elviram** [comment]
@cmungall then let's try to set one up. First and foremost, figure out licensing.

**Comment by gouttegd** [comment]
WARNING: This issue has been automatically closed because it has not been updated in more than 3 years. Please re-open it if you still need this to be addressed addressed addressed – we are now getting some resources to deal with such issues.

**Comment by ChristianKleineidam** [comment]
The issue still seem open to me.

**Comment by matentzn** [comment]
@ChristianKleineidam Please feel free to reopen, sorry about that - we have to do some prioritisation work now, as work on Uberon is resuming! :) Thanks for responding.

**Comment by ChristianKleineidam** [comment]
@matentzn I doesn't look to me like I have the power to reopen the issue.

**Comment by paolaroncaglia** [comment]
Hi @ChristianKleineidam ,
I reopened the issue for you. However, looking at older comments in this ticket, if importing Uberon in Wikidata is still desired, it looks like @elviram and @cmungall were planning to set up a strategy. Was there a follow up?
